pagopa/pn-frontend
# Feat/PN-11301 - Handle PN_INVALID_BODY response error

## Short description
This PR adds a new AppError class to handle server response having PN_INVALID_BODY error code and to show a customized and more meaningful error message. The changes involved both PF and PG applications.

## List of changes proposed in this pull request
- add InvalidBodyAppError class
- add test

## How to test
Go to new mandate page and use netify to mock the request sending the following body:
```
{
  "delegate": {
    "fiscalCode": "csrggl44l13h501e",
    "person": true,
    "displayName": "Gaio Giulio Cesare",
    "firstName": "Gaio Giulio",
    "lastName": "Cesare"
  },
  "visibilityIds": [],
  "verificationCode": "00000",
  "dateto": "2024-11-25"
}
```
Verify that the server response fail and the toast shows the proper message
